cmake/SociConfig.cmake

```diff
@@ -69,7 +69,7 @@ else()
             set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
         elseif (CMAKE_SYSTEM_NAME MATCHES "Windows")
             # We need to disable the warnings for using %I64d with MinGW.
-            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic-ms-format")
+            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic-ms-format -Wno-format")
```

Comment:
On Fri, Sep 18, 2015 at 08:28:42AM -0700, VZ wrote:
> > @@ -69,7 +69,7 @@ else()
> >              set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
> >          elseif (CMAKE_SYSTEM_NAME MATCHES "Windows")
> >              # We need to disable the warnings for using %I64d with MinGW.
> > -            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic-ms-format")
> > +            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic-ms-format -Wno-format")
> 
> Why do we need `-Wno-format`? I thought it compiled fine now, disabling the format strings would be really unfortunate...

Also, using %I64d is only valid for 32bit mode at least for MinGW64. It
would be nice to just depend on C++11 for Windows...

Joerg